config/locales/uk.yml

```diff
@@ -35,14 +35,11 @@ uk:
     contextual_sidebar:
       pretitle:
     cookie_banner:
-      buttons:
```

Comment:
It looks like you've deleted some text from the cookie banner here? That seems wrong, or have I misunderstood?

Reply:
Ah no sorry it's a mistake - i've now restored it.